lib/talon/resource.ex

```diff
@@ -226,6 +218,11 @@ defmodule Talon.Resource do
         Talon.Resource.name_field @__module__
       end
 
+      @spec resource_title(Module.t) :: String.t
+      def resource_title(resource) do
+        resource |> Map.get(Talon.Resource.name_field(resource.__struct__))
```

Comment:
This does not conform to the style guide. It should be replaced with `Map.get(resource, Talon.Resource...)`